src/tests/Makefile.am

```diff
@@ -7,6 +7,7 @@ NULL =
 EXTRA_DIST =                                                                   \
 	test_polkitd.py                                                        \
 	integration-test                                                       \
+	dbus-tests                                                             \
```

Comment:
Is this really adding the directory with all its contents?

Reply:
Documentation says it does -- https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html -- and it really adds all files to the archive.